main.py

```diff
@@ -11,25 +12,63 @@
 from myair import MyAirConnector
 
 logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.INFO)
+CONFIG_FILE='config.toml'
+
+def create_config():
+    config_data = {
+        "resmed": {
+            "login": os.getenv('RESMED_LOGIN'),
+            "password": os.getenv('RESMED_PASSWORD'),
+            "region": os.getenv('RESMED_REGION'),
+            "max_days": int(os.getenv('RESMED_MAX_DAYS', '365'))  # Default to 365 if not set
+        },
+        "influx": {
+            "url": os.getenv('INFLUX_URL'),
+            "bucket": os.getenv('INFLUX_BUCKET'),
+            "measurement": os.getenv('INFLUX_MEASUREMENT'),
+            "token": os.getenv('INFLUX_TOKEN'),
+            "org": os.getenv('INFLUX_ORG')
+        },
+        "main": {
+            "logverbosity": os.getenv('MAIN_LOGVERBOSITY', 'INFO'),  # Default to INFO if not set
+            "loop_minutes": int(os.getenv('MAIN_LOOP_MINUTES', '60'))  # Default to 60 if not set
+        }
+    }
+    with open(Path(__file__).with_name(CONFIG_FILE), "w") as config_file:
+        tomllib.dump(config_data, config_file)
```

Comment:
Would this end up deleting an existing config? Not good if someone didn't take a backup of their config and updates to new version of container.